protocol/0040-ASSF-asset_framework.md

```diff
@@ -212,6 +220,20 @@ NOTE 2: Before running this function, the user must run the ERC-20-standard `app
 
 This section will be expanded if additional ethereum based token standards are supported by Vega. New bridges will be expected to implement `IVega_Bridge`.
 
+#### Built-In Assets
+
+Built-in assets work much the same as any Vega asset except they can not be deposited or withdrawn. They also optionally can be restricted from use in specific functions. These functions include:
+
+- transfers
+- as the base asset in a spot market
+- as the quote asset in a spot market
+- as the settlement asset in a derivatives market (future or perpetual)
+
+These restrictions can be updated at any point through governance. When updating an asset the following cases should be considered:
+
+- given an asset allows transfers, if a delayed one-off or recurring transfer is created and then the asset is updated to not allow transfers, any outstanding transfers are not cancelled and are executed as normal.
+- given an asset allows use as a base, quote, or settlement asset, if a market is created using the asset and the asset is later updated to not allow the respective use, the market is not cancelled and should operate as normal.
+
```

Comment:
Now addressed in commit [269534d](https://github.com/vegaprotocol/specs/pull/2319/commits/269534d34d4fd3f7412a56493655be32e0491d95https://github.com/vegaprotocol/specs/pull/2319/commits/269534d34d4fd3f7412a56493655be32e0491d95)...

- New asset restriction added to restrict minting through a faucet.
- ELS assets default to restricting minting through a faucet.

Can update if we go down the new asset route rather than using built-in assets.